images.CI/download-customrepo.ps1

```diff
@@ -0,0 +1,14 @@
+param(
+    [String] [Parameter (Mandatory=$true)] $CustomRepoUrl,
```

Comment:
Let's get rid of `Custom` in parameters naming. Also I suggest to rename script to `download-repo.ps1`

Reply:
Done. Thanks!